### PR TITLE
Link Adapter Dev Guide from istio.io/refs/adapter

### PR DESCRIPTION
**Pull request by guptasu**

### CODE REVIEW

File: _docs/reference/config/mixer/adapters/index.md [hunk 1]
```diff
@@ -10,3 +10,5 @@ type: markdown
 
 {% include section-index.html %}
 
+To implement a new Adapter for Mixer, please refer to [Adapter Developer Guide](https://github.com/istio/mixer/blob/master/doc/dev/adapters.md)
```
Comment (douglas-reid) [inline _docs/reference/config/mixer/adapters/index.md]: nit: please refer to *the* Adapter Developer Guide

**Comment by guptasu** [comment]
@douglas-reid PTAL

File: _docs/reference/config/mixer/adapters/index.md [hunk 2]
```diff
@@ -10,3 +10,5 @@ type: markdown
 
 {% include section-index.html %}
 
+To implement a new Adapter for Mixer, please refer to the [Adapter Developer Guide](https://github.com/istio/mixer/blob/master/doc/dev/adapters.md)
```
Comment (geeknoid) [inline _docs/reference/config/mixer/adapters/index.md]: Adapter -> adapter
Developer -> Developer's
Add a missing period at the end.

File: _docs/reference/config/mixer/adapters/index.md [hunk 3]
```diff
@@ -10,5 +10,5 @@ type: markdown
 
 {% include section-index.html %}
 
-To implement a new Adapter for Mixer, please refer to the [Adapter Developer Guide](https://github.com/istio/mixer/blob/master/doc/dev/adapters.md)
+To implement a new Adapter for Mixer, please refer to the [adapter Developer's Guide](https://github.com/istio/mixer/blob/master/doc/dev/adapters.md).
```
Comment (geeknoid) [inline _docs/reference/config/mixer/adapters/index.md]: Sorry, wrong A capitalization:

new adapter
Adapter Developer's Guide

**Pull request merged**